### ISSUE TITLE
"@latest" in npm-shrinkwrap.json skips shrinkpack

### ISSUE THREAD

**Issue by JamieMason** [JamieMason/shrinkpack#54]
#### Expected behaviour

`npm install --loglevel http` should report no network activity when installing a shrinkpacked project.
#### Actual behaviour

Network activity is reported in some circumstances (see below).

```
npm http request GET http://registry.npmjs.org/options
npm http 304 http://registry.npmjs.org/options
```
#### Steps to reproduce

Have a dependency in `npm-shrinkwrap.json` which has a `from` containing `@latest` such as;

``` json
"options": {
  "version": "0.0.6",
  "from": "options@latest",
  "resolved": "file:node_shrinkwrap/options-0.0.6.tgz"
}
```
#### Software versions used

```
OS         : osx mavericks 
node.js    : 6.2.2
npm        : 3.10.4
shrinkpack : 0.12.5
```

**Comment by JamieMason** [comment]
`@latest` seems to not be relevant after all, removing all `"from"` entries from `npm-shrinkwrap.json` has no effect.

``` json
"options": {
  "version": "0.0.6",
  "resolved": "file:node_shrinkwrap/options-0.0.6.tgz"
},
```

**Comment by JamieMason** [comment]
Unable to reproduce.
